### ISSUE TITLE
npm install fail

### ISSUE THREAD

**Issue by amarant** [spatialillusions/milsymbol#20]
Hi,
it seems that npm install of milsymbol fail (on windows) with this error :
194 error enoent ENOENT: no such file or directory, chmod '<redacted>\node_modules\mil-symbol\MilSymbol.js'

Thanks.

**Comment by spatialillusions** [comment]
Hi, 

There has been a lot of changes in the new version of milsymbol, and you should now be able to install it using npm milsymbol.
I forgot to update the npm regrestry when I released the new version this week, but hopefully everything should be up and running now. I have been kind in of a rush since I’m going to present the library to US/NATO representatives next week. 

Please check out the new API documentation located in the library or at http://spatialillusions.com/milsymbol/

I hope that this hasn’t caused you any major problems. 

Regards
Måns Beckman

www.spatialillusions.com

**Comment by amarant** [comment]
So I tried the milsymbol package (why change the name from mil-symbol ?) and it still doesn't work, with the same error. I looked into it and the bin entry create the error, in the latest pull request I removed it and it works now.

**Comment by spatialillusions** [comment]
Hi, 

Sorry my fault, but it’s working now. 

The name change was to create consistancy, before the library was named MilSymbol, camel case, the file was named milsymbol, all lower case, and the npm was named mil-symbol with a dash in it. Now it is all just milsymbol, I understand that this might cause some troubles at the moment, but I think it is a change that makes things more future proof. I decided to do all this at the same time as I developed the new API, I didn’t like to break the API either, but I had a lot of request that I couldn’t make with the old API.
It is quite easy to port old code the new API, and the new API gives you so much more flexibility and consistency. (And all code behind is so much cleaner now, before I was kind of confused by it my self, it was a conglomerate showing how I added code to my base that I developed when I started building milsymbol for a basic demo, now it’s much cleaner code and it’s possible to manage the code in a much better way.)

Regards
Måns
